inst/include/population_dynamics/population/population.hpp

```diff
@@ -60,15 +60,12 @@ namespace fims
 
     // parameters are estimated; after initialize in create_model, push_back to parameter list - in information.hpp (same for initial F in fleet)
     std::vector<Type> log_naa;   /*!< estimated parameter: log numbers at age*/
-    std::vector<Type> log_Fmort; /*!< estimated parameter: log Fishing mortality*/
     std::vector<Type> log_M;     /*!< estimated parameter: log Natural Mortality*/
-    std::vector<Type> log_q;     /*!< estimated parameter: log catchability*/
+ 
 
     // Transformed values
     std::vector<Type> naa;   /*!< transformed parameter: numbers at age*/
-    std::vector<Type> Fmort; /*!< transformed parameter: Fishing mortality*/
     std::vector<Type> M;     /*!< transformed parameter: Natural Mortality*/
-    std::vector<Type> q;     /*!< transformed parameter: catchability*/
 
     std::vector<Type> ages;        /*!< vector of the ages for referencing*/
     std::vector<Type> mortality_F; /*!< vector of fishing mortality by year and age*/
```

Comment:
improve documentation to indicate this is population level fishing mortality summed across fleets